tests/feature_test.py

```diff
@@ -3,34 +3,64 @@
+bgp         enabled   enabled                                                                                        none
+database    enabled   disabled                                                                                       none
+dhcp_relay  enabled   enabled                                                                                        none
+lldp        enabled   enabled                                                                                        none
+nat         enabled   enabled                                                                                        none
+pmon        enabled   enabled                                                                                        none
+radv        enabled   enabled                                                                                        none
+restapi     disabled  enabled                                                                                        none
+sflow       disabled  enabled                                                                                        none
+snmp        enabled   enabled                                                                                        none
+swss        enabled   enabled                                                                                        none
+syncd       enabled   enabled                                                                                        none
+teamd       enabled   enabled                                                                                        none
+telemetry   enabled   enabled                                                                                        none
+"""
+
+show_feature_config_output="""\
+Feature     State     AutoRestart    Owner    no-fallback
+----------  --------  -------------  -------  -------------
+bgp         enabled   enabled        local    false
+database    enabled   disabled       local    false
+dhcp_relay  enabled   enabled        kube     false
+lldp        enabled   enabled        kube     false
+nat         enabled   enabled        local    false
+pmon        enabled   enabled        kube     false
+radv        enabled   enabled        kube     false
+restapi     disabled  enabled        local    false
+sflow       disabled  enabled        local    false
+snmp        enabled   enabled        kube     false
+swss        enabled   enabled        local    false
+syncd       enabled   enabled        local    false
+teamd       enabled   enabled        local    false
+telemetry   enabled   enabled        kube     false
 """
 
 show_feature_bgp_status_output="""\
-Feature    State    AutoRestart
----------  -------  -------------
-bgp        enabled  enabled
+Feature    State    AutoRestart    SystemState    UpdateTime    ContainerId    ContainerVersion    CurrentOwner    RemoteState
```

Comment:
i think we should only show these data when kubernates is enabled. we should not show these column if kubernates feature is not enabled.

Reply:
Sure.

Reply:
Shows only the available fields.